Cronos/cronos-evm-starter-via-rpc/src/mappings/mappingHandlers.ts

```diff
@@ -38,8 +37,8 @@ export async function handleEthermintEvmCall(
   const approval = Approval.create({
     id: call.hash,
     owner: call.from,
-    value: call.args._value.toBigInt(),
-    spender: call.args._spender,
+    value: call.args[1].toBigInt(),
```

Comment:
Same deal here https://github.com/subquery/cosmos-subql-starter/blob/main/Cronos/cronos-evm-starter-via-rpc/erc20.abi.json#L110

I'm not sure where this abi came from